pipelines/src/pipelines/trigger/main.py

```diff
@@ -191,8 +190,6 @@ def sandbox_run(args: List[str] = None) -> aiplatform.PipelineJob:
     Returns the PipelineJob object of the triggered pipeline run.
     Usage: python main.py --template_path=pipeline.json --enable_caching=true
     """
-    logging.basicConfig(level=logging.DEBUG)
-
```

Comment:
could also do: 

```python
logging.getLogger(__name__). setLevel(logging.DEBUG)
```

Reply:
Debug is overkill I think, probably best to just remove the logging config from here